lib/platform/node/node_http.js

```diff
@@ -65,7 +69,18 @@
                 
                 // Note that we were aborted
                 req.wasAborted = true;
-            });
+            }
+
+            // req.on("abort", function() {
+            //     var res = { headers: {}, statusCode: "abort" };
+            //     var data = "{}";
+            //     var complete_response = that._buildResponse("abort", res, data);
+                
+            //     callback(complete_response);
+                
+            //     // Note that we were aborted
+            //     req.wasAborted = true;
+            // });
```

Comment:
I think we can remove this

Reply:
Commented section has been removed as suggested.